app/views/applications/deploy.html.erb

```diff
@@ -0,0 +1,93 @@
+<% content_for :page_title, @application.name %>
+
+<div class="page-header">
+  <h1 class="remove-bottom-margin">
+    <span class="name">Deploy <%= @application.name %></span>
+    <span class="shortname">(<%= @application.shortname %>)</span>
+  </h1>
+  <%= link_to @application.repo_url, @application.repo_url, target: "_blank" %>
+</div>
+
+<main>
+  <h2>Candidate Release <span class="label label-info"><%= @release_tag %></span></h2>
+  <p class="lead add-top-margin">Production was deployed to <span class="label label-danger"><%= @production_deploy.version %></span><%= time_ago_in_words(@production_deploy.created_at) %> ago</p>
```

Comment:
You're right, that reads badly. Leading with the tag doesn't read quite right either, how about: 

> Production is release_2156 — deployed at 11:59am on Monday

(I've also changed the date/time format to have more granularity and match the application list.)